replica/database.cc

```diff
@@ -1756,6 +1756,50 @@ future<> database::apply_with_commitlog(column_family& cf, const mutation& m, db
     }
 }
 
+future<> database::apply(const std::vector<frozen_mutation>& muts, db::timeout_clock::time_point timeout) {
```

Comment:
Let's call it apply_in_memory() (like the existing one) so people aren't tempted to think it is durable.

Reply:
yes, doh